### DISCUSSION TITLE
Firmware 4.0 Disconnect Stream Using S63=2 for DTR Function

### DISCUSSION THREAD

**Discussion by shawnmerrick** [bozimmerman/Zimodem#143] [Q&A]
@bozimmerman Since 1986 my BBS used drop of DTR line to disconnect a caller.  This worked fine on firmware versions PRIOR to 4.0

I did a lot of checking on 4.0 to find there is a new S63 register which defaults to 0 but ensures the DTR disconnect when set to 2.  I'm wondering if this is now the only way to do so, and whether it will negatively affect anything else.

Your insight would be greatly appreciated.

     Shawn

**Comment by bozimmerman** [comment]
So, version 3.7.1 of the firmware is still available in its own branch, and is recommended for the ESP8266.

If you intend to remain with 4.0, then yes, the S63 register is the one to go.  It does save with your configuration with AT&W though, so you only have to set it once.
